### PR TITLE
adding tooltip for category description

### PR DESCRIPTION
**Pull request by Jbarget**

### CODE REVIEW

File: client/src/css/animations/_category-info.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  position: absolute;
+  padding: var(--spacing-small);
+  right: var(--spacing-small);
+  font-size: var(--font-size-7);
+  background: var(--white);
+  width: 70%;
+  z-index: 1;
+  -webkit-box-shadow: 2px 2px 11px 0px rgba(0,0,0,0.75);
+  -moz-box-shadow: 2px 2px 11px 0px rgba(0,0,0,0.75);
+  box-shadow: 2px 2px 11px 0px rgba(0,0,0,0.75);}
```
Comment (mantagen) [inline client/src/css/animations/_category-info.css]: 0.25 looks a little softer on the eye imo

File: client/src/css/animations/_category-info.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    to {opacity: 0;}
+}
+
+@keyframes fadeout {
+    from {opacity: 1;}
+    to {opacity: 0;}
+}
+
+.category-info {
+  visibility: hidden;
```
Comment (mantagen) [inline client/src/css/animations/_category-info.css]: i found that `left: 4em` looks good on this ... might be a var i don't know

Comment (Jbarget) [reply client/src/css/animations/_category-info.css]: i dont think we're seeing the same thing, i apply that and i get 

screen shot 2018-08-30 at 18 03 32

Comment (Jbarget) [reply client/src/css/animations/_category-info.css]: Went with `left: 7em;` let me know how it looks for you


screen shot 2018-08-31 at 09 30 01

**Comment by Jbarget** [comment]
also i added `pointer-events: none` so that the tooltips' hover state is ignored

File: client/src/css/animations/_category-info.css (hunk 2 above)
Comment (mantagen) [reply client/src/css/animations/_category-info.css]: :+1:

**Comment by mantagen** [comment]
@Jbarget , nice, conflicts then this can go in

**Pull request merged**